src/Json.Schema.Validation/RuleResources.resx

```diff
@@ -118,68 +118,68 @@
   </data>
   <data name="ErrorInvalidTypeString" xml:space="preserve">
-    <value>"{1}" is not a valid schema type.</value>
+    <value>'{1}' is not a valid schema type.</value>
   </data>
   <data name="ErrorInvalidTypeType" xml:space="preserve">
-    <value>JSON schema requires the value of "type" to be either a string or an array of strings, but a token of type "{1}" was seen.</value>
+    <value>JSON schema requires 'type' to be either a string or an array of strings, but a token of type '{1}' was seen.</value>
   </data>
   <data name="ErrorMessageStringWithPath" xml:space="preserve">
-    <value>at "{{0}}": {0}</value>
+    <value>{{0}}: {0}</value>
   </data>
   <data name="ErrorNotAllOf" xml:space="preserve">
-    <value>The instance does not validate successfully against all of the {1} schemas specified by "allOf".</value>
+    <value>The instance does not validate successfully against all of the {1} schemas specified by 'allOf'.</value>
   </data>
   <data name="ErrorNotAMultiple" xml:space="preserve">
-    <value>The value {1} is not a multiple of {2}.</value>
+    <value>{1} is not a multiple of {2}.</value>
   </data>
   <data name="ErrorNotAnyOf" xml:space="preserve">
-    <value>The instance does validate successfully against any of the {1} schemas specified by "anyOf".</value>
+    <value>The instance does validate successfully against any of the {1} schemas specified by 'anyOf'.</value>
   </data>
   <data name="ErrorNotAString" xml:space="preserve">
-    <value>The property is required to be a string, but the token is of type "{1}".</value>
+    <value>The property is required to be a string, but the token is of type '{1}'.</value>
   </data>
   <data name="ErrorNotOneOf" xml:space="preserve">
-    <value>The instance validates successfully against {1} of the {2} schemas specified by "oneOf", instead of validating successfully against exactly one of them.</value>
+    <value>The instance validates successfully against {1} of the {2} schemas specified by 'oneOf', instead of validating successfully against exactly one of them.</value>
   </data>
   <data name="ErrorNotUnique" xml:space="preserve">
     <value>The elements of the array are not unique.</value>
   </data>
   <data name="ErrorRequiredPropertyMissing" xml:space="preserve">
-    <value>The required property "{1}" is missing.</value>
+    <value>The required property '{1}' is missing.</value>
   </data>
   <data name="ErrorStringDoesNotMatchPattern" xml:space="preserve">
-    <value>The string "{1}" does not match the regular expression "{2}".</value>
+    <value>'{1}' does not match the regular expression '{2}'.</value>
   </data>
   <data name="ErrorStringTooLong" xml:space="preserve">
-    <value>The string "{1}" has length {2}, which is greater than the maximum length {3}.
+    <value>'{1}' has length {2}, which is greater than the maximum length {3}.
```

Comment:
maximum length as specified in the schema. 'maximum specified length'. could be clearer. i don't generally favor more verbose reporting but am on the fence on this. in this example, btw, i see why you are not enquoting literal numeric values. that's probably right.